worlds.md

```diff
@@ -54,4 +54,4 @@
  2019-09-20 |    6    |     76     | [Barasii](https://barasii.github.io/a-tiny-JS-world/)
  2019-10-26 |    4    |     39     | [karelskiy](https://karelskiy.github.io/a-tiny-JS-world/)
  2019-11-02 |    5    |     37     | [dashakim](https://dashakim.github.io/a-tiny-JS-world/) 
- 
+ 2019-11-19 |    5    |     50      | [dafen173](https://github.com/dafen173/a-tiny-JS-world)
```

Comment:
Please ensure file ends with an empty line.
It is a good style to have one always.
The most of IDEs have a relevant setting so you can take the effort once. Normally it is something like "ensure empty line at the file end"